atala-prism-sdk/src/commonMain/kotlin/io/iohk/atala/prism/walletsdk/domain/models/keyManagement/ExportableImportableKey.kt

```diff
@@ -0,0 +1,77 @@
+package io.iohk.atala.prism.walletsdk.domain.models.keyManagement
+
+import io.iohk.atala.prism.apollo.base64.base64UrlDecoded
+import io.iohk.atala.prism.apollo.base64.base64UrlDecodedBytes
+import io.iohk.atala.prism.apollo.base64.base64UrlEncoded
+import kotlinx.serialization.Serializable
+
+interface ExportableKey {
+    fun getPem(): String
+    fun getJwk(): JWK
+
+    fun jwkWithKid(kid: String): JWK
+}
+
+interface ImportableKey {
+    @Throws(Exception::class)
+    fun initializeFromPem(pem: String)
+
+    @Throws(Exception::class)
+    fun initializeFromJwk(jwk: JWK)
+}
+
+@Serializable
+data class JWK(
+    val kty: String,
+    val alg: String? = null,
+    val kid: String? = null,
+    val use: String? = null,
+    val n: String? = null,
+    val e: String? = null,
+    val d: String? = null,
+    val p: String? = null,
+    val q: String? = null,
+    val dp: String? = null,
+    val dq: String? = null,
+    val qi: String? = null,
+    val crv: String? = null,
+    val x: String? = null,
+    val y: String? = null,
+    val k: String? = null
+)
+
+data class PEMKey(val keyType: String, val keyData: ByteArray) {
```

Comment:
Can we change the `keyType` to be an enumeration as we have many types for the keys type, and they must be in a specific format 
ex: `-----BEGIN RSA PRIVATE KEY-----`, `-----BEGIN RSA PUBLIC KEY-----`, `-----BEGIN PRIVATE KEY-----`, `-----BEGIN PUBLIC KEY-----`, etc.

The current implementation is for typical keys (Public and Private keys) without any additional information.
Also, so that users of the `PEMKey` can always get a valid encoded `PEM`.